DirectX11/globals.h

```diff
@@ -11,13 +11,21 @@
 
 #include "DLLMainHook.h"
 #include "DirectXMath.h"
-#include "util.h"
 #include "DecompileHLSL.h"
 
 #include "ResourceHash.h"
 #include "CommandList.h"
 #include "profiling.h"
 #include "lock.h"
+#include <enum.h>
+
+#include "../HackingCommon/globals_common.h"
+#define ResourceSnapshot Snapshot<ID3D11Resource>
+#define ShaderInfoData InfoData<ID3D11Resource>
+#define TargetResourceSelection TargetResourceSelection<ID3D11Resource*, ID3D11Resource, ResourceHashInfo>
+#define ShaderResourceSelection_(type) ShaderResourceSelection<UINT64, ID3D11Resource, type>
+
+using namespace std;
```

Comment:
using namespace std; is considered bad practice, and should especially be avoided in header files. I know we have used it in a few other places in 3DMigoto, but if you don't need it, don't add it. Certainly don't add it here.

https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice